<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiGrabber 1.0 by zaza</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Keyword</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Keyword:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEdit_keyword">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_title">
               <property name="text">
                <string>Title</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Shops</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_otto">
            <property name="text">
             <string>Otto.de</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_mediamarkt">
            <property name="text">
             <string>MediaMarkt.de</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_fahrrad">
            <property name="text">
             <string>FahrradXXL.de</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Url</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Url:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_url"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Single</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Url:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="lineEdit_singleUrl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_settings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_min_price_filter"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_captcha_api_key"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_max_price_reduction"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Price reduction (%)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEdit_max_price_filter"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Price filter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Max. Images per product</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max. Products per Keyword</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>2Captcha Api Key</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_max_images"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_max_products"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_min_price_reduction"/>
      </item>
      <item row="5" column="3">
       <widget class="QRadioButton" name="option_99">
        <property name="text">
         <string>Round to ,99</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="option_00">
        <property name="text">
         <string>Round to ,00</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Rounding</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_shop">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Shop</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ShopType</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="option_prestashop">
          <property name="text">
           <string>Prestashop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="option_shopware">
          <property name="text">
           <string>Shopware</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_shopware" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Shopware Admin Username</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_shopwareAdmin"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Shopware Api Key</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_shopwareApiKey"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_shopwareURL"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Shopware Shop URL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_prestashop" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Prestashop Api Key</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_prestaApiKey"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Prestashop URL</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_prestaURL"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_stop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clearLog">
       <property name="text">
        <string>Clear Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_log">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
